docs/mkdocs/documentation/configure.md

```diff
@@ -1,15 +1,44 @@
 # Configuring
 
-KMM should be configured out of the box with sensible defaults.
-The operator configuration is set in the `kmm-operator-manager-config` `ConfigMap` in the operator namespace.
-To modify any setting, edit the `ConfigMap` data and restart the controller with the following command:
+KMM is configured out of the box with sensible defaults.
+The operator configuration is set in `internal/config/config.go` in `newDefaultConfig()`.
+To modify any setting, create a `ConfigMap` with name of `kmm-operator-manager-config` in the operator namespace with 
+the relevant data and restart the controller with the following command:
 
 ```shell
 kubectl delete pod -n "$namespace" -l app.kubernetes.io/component=kmm
 ```
 
 The value of `$namespace` depends on your [installation method](install.md).
 
+## Example
+```yaml
+apiVersion: v1
+data:
+  controller_config.yaml: |
+    healthProbeBindAddress: :8081
+    webhookPort: 9443
+    leaderElection:
+      enabled: true
+      resourceID: kmm.sigs.x-k8s.io
+    metrics:
+      enableAuthnAuthz: true
+      bindAddress: 0.0.0.0:8443
+      secureServing: true
+    worker:
+      runAsUser: 0
+      seLinuxType: spc_t
+      firmwareHostPath: /example/firmware
+kind: ConfigMap
+metadata:
+  name: kmm-operator-manager-config
+  namespace: kmm-operator-system
+```
```

Comment:
Let's give an example that uses non-default values (We can only change the `worker.firmwareHostPath` for example)